charts/artifact-caching-proxy/templates/nginx-proxy-configmap.yaml

```diff
@@ -61,6 +71,10 @@ data:
 
         # Default location for cached requests
         location / {
+{{- if .Values.proxy.proxyBypass.enabled -}}
+            add_header Cache $upstream_cache_status;
```

Comment:
The `Cache` header is, of course, optional but I like the transparency it provides.